### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by jmyers0022**

### CODE REVIEW

File: app/assets/stylesheets/nav.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.sign-in-form {
+  padding: 10px;
+
+  li:hover {
+    background-color: inherit !important;
+  }
+}
+
+.forgot-password-link {
+  color: #999;
```
Comment (KMyers44) [inline app/assets/stylesheets/nav.scss]: You could probably put that color in a sass variable

File: app/views/shared/_nav.html.erb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<ul id="sign-in-dropdown" class="js-sign-in-form sign-in-form dropdown-content">
+  <li>
+    <%= link_to 'Sign In', '#', :class => 'sign-in' %>
```
Comment (KMyers44) [inline app/views/shared/_nav.html.erb]: The default should be prevented

File: app/assets/stylesheets/nav.scss (hunk 1 above)
Comment (jmyers0022) [inline app/assets/stylesheets/nav.scss]: We'll put variables in when we start repeating colors

File: app/views/shared/_nav.html.erb (hunk 2 above)
Comment (jmyers0022) [inline app/views/shared/_nav.html.erb]: Looks like materialize is already preventing the default

**Pull request merged**